### ISSUE TITLE
Negating Numbers, and Logical Not

### ISSUE THREAD

**Issue by Magnogen** [TodePond/DreamBerd#4]
`;` is a logical not, and traditionally, negating numbers uses a `-`. It's sort of a similar thing, right?

- They're both reversible (`;;a == a` && `--a == a`)
- They also both have one to one mapping (`a` only has one value for `-a`, and vice versa. `a` only has one value for `;a`, and vice versa)

Why not just remove the `-`, and make negative 3 be described with `;3`?

It also makes a nice winking face ;3

**Comment by Magnogen** [comment]
Precisely, any symbols that _are_ overloaded should have reasonable logic behind it. And `;3` is reason enough.

**Comment by berrymot** [comment]
then how do we concatenate strings? `+`

**Comment by TodePond** [comment]
We can't use `+` because that's add.
We can't use `++` because that's increment.

I guess we'll have to use `+++`.

**Comment by Magnogen** [comment]
What about using an `&`?

```c
const const name = "Lu"!
"Hello, " & name & "!"? // Hello, Lu!
```

It's kind of makes sense from a reading englishy standpoint, but I'm not sure what your stance is on logical and - is that `&&`, like JavaScript?

**Comment by jasonflorentino** [comment]
if & is to be used for string concatenation, I'm curious to hear people's thoughts on bitwise and. Are the operators just the words `and`, `or`, `xor`, etc.? Or perhaps to be extra clear that these are for bits only and not for logic, there is a `b` in front of them as in `band`, `bor`, `bxor` and so on?

**Comment by TodePond** [comment]
Maybe binary and should be `01100001 01101110 01100100`?

**Comment by Joshix-1** [comment]
> So we must not overload the - operator. Currently it does subtracting AND inverting numbers. 

Why do we need an operator for subtraction? Isn't subtraction just adding an inverted number?

**Comment by ThePython10110** [comment]
Good point. And since `-` wouldn't indicate subtraction, the increment/decrement operators should be `↑` and `↓`, not `--`. With that in mind, what should we do with the `-` character?
